angular/playground/projects/ng-entity-service/src/lib/ng-entity-service-Notifier.ts

```diff
@@ -0,0 +1,38 @@
+import { Subject } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { filter } from 'rxjs/operators';
+import { Msg } from './types';
+
+export enum HttpMethod {
```

Comment:
any reason not using a const enum ?

Reply:
There are not included in the bundle. Users will not be able to consume them.